www/functions/createCheckboxQuestion.R

```diff
@@ -8,9 +8,13 @@ createCheckboxQuestion <- function(questionId, choiceNames, choiceValues, select
   
   choiceNames <- lapply(choiceNames, function(x) HTML(x))
   
+  if (!is.null(questionLabel)){
```

Comment:
@karolkiljan a co się dzieje jak questionLabel jest NULLem i damy go do tej funkcji HTML(). Error?

Reply:
Nie. Tworzy własnie ten pusty label, który przepycha pierwszą kolumnę checkboxów w dół

Reply:
Aa, czyli dlatego się to tak rozeszło. Oki